src/bokeh_fastapi/handler.py

```diff
@@ -120,7 +120,7 @@ async def get(
         self, request: Request, bokeh_session_id: Optional[ID] = None
     ) -> HTMLResponse:
         session = await self.get_session(request, bokeh_session_id)
-        page = self.render_session(
+        page = DocHandler.render_session(
```

Comment:
Should this be

```suggestion
        page = type(self).render_session(
```

?